source/minio-server/minio-server.rst

```diff
@@ -321,3 +327,36 @@ Root Credentials
    the server configuration with the new credentials. After the process
    restarts successfully, you can restart it without 
    :envvar:`MINIO_SECRET_KEY_OLD`.
+
+Storage Class
+~~~~~~~~~~~~~
+
+These environment variables configure the :ref:`parity <minio-ec-parity>`
+to use for objects written to the MinIO cluster. 
+
+MinIO Storage Classes are distinct from AWS Storage Classes, where the latter
+refers to the specific storage tier on which to store a given object. 
+
+.. envvar:: MINIO_STORAGE_CLASS_STANDARD
+
+   The number of :ref:`parity blocks <minio-ec-parity>` to create for 
+   objects with the standard (default) storage class. MinIO uses the
+   ``EC:N`` notation to refer to the number of parity blocks (``N``).
+   This environment variable only applies to deployments with 
+   :ref:`Erasure Coding <minio-erasure-coding>` enabled. 
+
+   Defaults to ``4``. 
+
+.. envvar:: MINIO_STORAGE_CLASS_REDUCED
+
+   The number of :ref:`parity blocks <minio-ec-parity>` to create for objects
```

Comment:
No comments!
I just wanted to include here a piece of text about possible redundancy use cases.
I saved it some time ago for myself. Just FYI.
```
However, high redundancy also means higher storage usage. There are certain cases where such high parity count and hence storage usage, may not be desirable, for example:
- Users may have deployed specialized storage hardware with fail safety. So, they aren’t expecting any failures or frequent failures.
- Data being stored is less critical, and/or of lesser value compared to storage value itself.
- Data in MinIO store is part of a backup/restore schema and archived regularly.
```

Reply:
Still helpful! I might integrate this overall into discussions on REDUCED_REDUNDANCY.